test/ex_webrtc/rtp/jitter_buffer/heap_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule ExWebRTC.RTP.JitterBuffer.HeapTest do
+  use ExUnit.Case, async: true
+
+  alias ExWebRTC.RTP.JitterBuffer.Heap
+
+  test "stores things and reports size" do
```

Comment:
Nitpick, in Elixir WebRTC we decided to avoid the old Membrane style of describing tests. Sometimes you can still find such tests or it might not be posssible to describe them in other way but in general we favor something like this:

```
test "Heap.push/1"
test "Heap.root/1"
test "Heap.new/1"
```

That's how tests in Elixir repo are written. We don't have to change these tests, just for the future